flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -167,6 +167,15 @@
 		.defaultValue("")
 		.withDescription("A comma-separated list of tags to apply to the Flink YARN application.");
 
+	/**
+	 * The address of Flink HistoryServer.
+	 * This configuration parameter allows setting the appTrackingUrl for finished YARN applications.
+	 */
+	public static final ConfigOption<String> APPLICATION_HISTORY_SERVER_ADDRESS =
```

Comment:
Should we rather use `HistoryServerOptions#HISTORY_SERVER_WEB_ADDRESS`?

Reply:
I have thought about that, but `HistoryServerOptions#HISTORY_SERVER_WEB_ADDRESS` is a hostname, what if we want to set a domain name instead?

_http://host:port vs http://domain-name_

Reply:
@tillrohrmann Any suggestion? Should we stick to `HistoryServer#HISTORY_SERVER_WEB_ADDRESS` ?

Reply:
I think we could start with `hostname:port` and extend it if we see that there is a problem with that. At the moment people would need to connect to the `HistoryServer` that way anyway.

Reply:
I'll apply the change and merge the PR.